Documentation/op-guide/configuration.md

```diff
@@ -289,6 +289,12 @@ Follow the instructions when using these flags.
 + Set level of detail for exported metrics, specify 'extensive' to include histogram metrics.
 + default: basic
 
+## Auth flags
+
+### --auth-token-opts
++ Specify a token type and token specific options, especially for JWT. Its format is "type,var1=val1,var2=val2,...". Possible type is 'simple' or 'jwt'. Possible variables are 'sign-method' for specifying a sign method of jwt (its possible values are 'ES256', 'ES384', 'ES512', 'HS256', 'HS384', 'HS512', 'RS256', 'RS384', 'RS512', 'PS256', 'PS384', or 'PS512'), 'pub-key' for specifying a path of public key for veryfing jwt, and 'priv-key' for specifying a path of private key for signing jwt.
```

Comment:
Can this have an example of a fully formed `jwt` option?

s/veryfying/verifying
s/a path of/a path to a/